user/enterprise/troubleshooting-guide.md

```diff
@@ -0,0 +1,160 @@
+
+
+## travis-worker on Ubuntu 16.04 does not start
+
+### The problem
+
+`travis-worker` was installed on a fresh installation of Ubuntu 16.04 (Xenial) with no issues. However, the command `sudo systemctl status travis-worker` shows that it is not running.
+
+In addition, the command `sudo journalctl -u travis-worker` contains the following error:
+
+```
+/usr/local/bin/travis-worker-wrapper: line 20: /var/tmp/travis-run.d/travis-worker: No such file or directory
+```
+
+### Strategy
+
+One possible reason that travis-worker is not running is that `systemctl` cannot create a temporary directory for environment files. To fix this, please create the directory `/var/tmp/travis-run.d/travis-worker` and assign write permissions via:
+
+```sh
+$ mkdir -p /var/tmp/travis-run.d/
+$ chown -R travis:travis /var/tmp/travis-run.d/
+```
+
+## Builds fail with curl certificate errors
+
+### The problem
+
+A build fails with a long `curl` error message similar to:
+
+```
+curl: (60) SSL certificate problem: unable to get local issuer certificate
+```
+
+This can have various causes, including an automatic nvm update or a caching error.
+
+### Strategy
+
+This error is most likely caused by a self-signed certificate. During the build, the worker container attempts to fetch different files from the platform machine. If the server was originally provisioned with a self-signed certificate, curl doesn't trust this certificate and therefore fails. While we're working on resolving this in a permanent way, currently the only solution is to install a certificate issued by a trusted Certificate Authority (CA). This can be a free Let's Encrypt certificate or any other trusted CA of your choice. We have a section in our [Platform Administration Tips](/user/enterprise/platform-tips/#use-a-lets-encrypt-ssl-certificate) page that walks you through the installation process using Let's Encrypt as an example.
+
+## User accounts are stuck in syncing state
+
+### The problem
+
+One or more user accounts are stuck in the `is_syncing = true` state. When you query the database, the number of users which are currently syncing does not decrease over the time. Example:
+
+```sql
+travis_production=> select count(*) from users where is_syncing=true;
+ count
+-------
+  1027
+(1 row)
+```
+
+### Strategy
+
+Log into the platform machine via ssh. You can reset the `is_syncing` flag for user accounts that are stuck by running:
+
+```bash
+$ travis console
+>> User.where(is_syncing: true).count
+>> ActiveRecord::Base.connection.execute('set statement_timeout to 60000')
+>> User.update_all(is_syncing: false)
+```
+
+It can happen that organizations are also stuck in the syncing state. Since an organization itself does not have a `is_syncing` flag, all users that do belong to it have to be successfully synced.
```

Comment:
I wanted to call out that I removed the "Contact Enterprise Support" section from this renamed file. Personally I am not a fan of having the same section included at the end of every page. I also find that it muddies up the generated table of contents at the top of each page.

I have a proposal for an overall change regarding that but I'll leave it for a separate PR so this one doesn't grow completely unmanageable. 😄 

For what its worth: not including that section matches our recent update to the [updated Enterprise Setup Page](https://docs.travis-ci.com/user/enterprise/setting-up-travis-ci-enterprise/).